app/Exceptions/Handlers/AccessDBDenied.php

```diff
@@ -40,7 +40,7 @@ public function renderHttpException(SymfonyResponse $defaultResponse, HttpExcept
 			$contentType = $defaultResponse->headers->get('Content-Type');
 			if (!empty($contentType)) {
 				$redirectResponse->headers->set('Content-Type', $contentType);
-				$redirectResponse->setContent($defaultResponse->getContent() ?: null);
+				$redirectResponse->setContent($defaultResponse->getContent() !== false ? $defaultResponse->getContent() : null);
```

Comment:
If `getContent` generates the content dynamically (i.e. a blade template), it might be more efficient not to call the method twice but cache the previous result:

```suggestion
				$content = $defaultResponse->getContent();
				$redirectResponse->setContent($content !== false ? $content : null);
```

I have also checked the signature of the methods `setContent` and `getContent` to verify, that we cannot simply use `setContent($defaultResponse->getContent())`. Indeed `setContent` requires a nullable string `?string` while `getContent()` returns `string|false`. This is appalling.